src/components/Header/index.tsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import Styles from "./Header.module.css";
+import NavBar from "../NavBar";
+import Link from "next/link";
+
+const Header = () => {
```

Comment:
the header is too simple and not scalable, I recommend adding a property for the list of links at least for example an array like that:
```javascript
const listOfLinks = [
  {label: home, url: '/'},
  {label: cars, url: '/cars'}
]
```

and them pass the list as a property on the `Header` component (use typescript for better scalability)